### PR TITLE
Identity/Securing privileged access: grammar

### LINKED ISSUE #5002: Grammar
Add "a" and "."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f43fec44-fdc4-3f92-291e-a46c10cd4aee
* Version Independent ID: 0ed1458c-2026-696f-a7d1-f964137e1377
* Content: [Securing privileged access](https://docs.microsoft.com/en-us/windows-server/identity/securing-privileged-access/securing-privileged-access)
* Content Source: [WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md)
* Product: **windows-server**
* Technology: **security**
* GitHub Login: @MicrosoftGuyJFlo
* Microsoft Alias: **joflore**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As reported in issue ticket #5002 (**Grammar**), some sentences are missing their ending period dot, some letters are missing or incorrect.

Thanks to Adrian Hale (@adrianhale) for reporting this grammar issue.

**Changes proposed:**

- add missing period dots at the end of some sentences
- remove incorrectly used article (a) where required
- add missing article "the" where required
- add missing readability commas where required
- rephrase some unclear or imprecise phrases
- add missing hyphens in "day-to-day" and "three-tiered"
- add the pronoun "that" where required

**Whitespace changes:**

- add editorial blank line between the metadata section and the page title line
- add MarkDown indent marker compatibility spacing before "Applies To:"

**Additional notes:**

- Some of my changes might be missing in the list above, but the gist is that all my proposed changes are of a grammatical nature.
- My English comprehension may be colored by British English, so if I have made a change not quite making sense in an American English text, feel free to correct me.

**Ticket closure or reference:**

Closes #5002

### CODE REVIEW

**Comment by PRMerger14** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — COMMENTED** [review]
**Rich Diff view:** https://github.com/MicrosoftDocs/windowsserverdocs/pull/5003/files?diff=unified&w=1&short_path=ecb9c13#diff-ecb9c13a9ef962acdb582d8e7613137171bb21facf773f5e7e5a07912f9a48a3

When using the rich diff view, for readability, to review the changes, please note that the Note blob paragraph change only contains added commas, but it is shown as containing a major rewrite. The Note blob starts with the following text:

> [!NOTE] Securing privileged access requires a broad range of elements

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@illfated: I'm sorry - only the author of this article, @MicrosoftGuyJFlo, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md [hunk 1]
```diff
@@ -9,43 +9,44 @@ author: MicrosoftGuyJFlo
...
 * Adversaries have demonstrated a consistent and ongoing ability to obtain access on workstations inside the network boundary through phishing and other web and email attacks.
 
-These factors necessitate building a modern security perimeter out of authentication and authorization identity controls in addition to the traditional network perimeter strategy. A security perimeter here is defined as a consistent set of controls between assets and the threats to them. Privileged accounts are effectively in control of this new security perimeter so it's critical to protect privileged access.
+These factors necessitate building a modern security perimeter out of authentication and authorization identity controls, in addition to the traditional network perimeter strategy. A security perimeter here is defined as a consistent set of controls between assets and the threats to them. Privileged accounts are effectively in control of this new security perimeter, so it is critical to protect privileged access.
```
Comment (eross-msft) [inline WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md:34]: @illfated  We actually use contractions in our content now as part of our updated voice and tone. If you could revert that single change back to it's, I'd appreciate it.

**Comment by PRMerger8** [comment]
@illfated : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md (hunk 1 above)
Comment (illfated) [reply WindowsServerDocs/identity/securing-privileged-access/securing-privileged-access.md]: Fair enough. I presume my suggestion of removing the contraction comes from my experience with British English.

Reverted, as requested.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@illfated: I'm sorry - only the author of this article, @MicrosoftGuyJFlo, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by eross-msft — APPROVED** [review]

**Pull request merged**